### ISSUE TITLE
Yaw angle array shape verification

### ISSUE THREAD

**Issue by carrommad** [NREL/floris#454]
**Bug description**
Typo in dimensions of yaw array inn line 73 of _01_opening_floris_computing_power.py_ :
> yaw_angles = np.zeros([1,3,2]) # 1 wind direction, 3 wind speeds, 2 turbines

**Expected behavior**
Shouldn't it be 3 wind directions?

**Screenshots, if applicable**
<Add screenshots to help explain your problem.>

**Floris Version**
v3.1.1

**Comment by carrommad** [comment]
Yes, exactly. Sorry for not pasting the link. Am I wrong about the typo then?

**Comment by misi9170** [comment]
I don't know that this was every fixed, but I think the issue is now stale and I'm going to close as incomplete.
